protocols/api/pom.xml

```diff
@@ -46,6 +46,10 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-handler</artifactId>
+        </dependency>
```

Comment:
I think this dependency was there on purpose, not to hard code the use of netty in the middle of protocols-api